docs/README.md

```diff
@@ -34,6 +34,38 @@ The user interface and content is available in both English and Spanish. Additio
 
 The application communicates with external services like [Littlepay][littlepay] via API calls and others like the [Identity Gateway](https://dev.auth.cdt.ca.gov) via redirects, both over the public internet. See [all the system interconnections][interconnections].
 
+## Security
+
+Cal-ITP takes security and privacy very seriously. Below is an overview of how the system is designed with security in mind.
+
+### Architecture
+
+The Benefits application is deployed to Microsoft Azure. Traffic is encrypted between the user and the application, as well as between the application and external integrations.
+
+The network is managed by the [California Department of Technology (CDT)](https://cdt.ca.gov/), who provide a firewall and [distributed denial-of-service (DDoS)](https://www.cloudflare.com/learning/ddos/what-is-a-ddos-attack/) protection.
+
+You can find more technical details on [our infrastructure page](deployment/infrastructure/).
+
+### Data storage
+
+The Benefits application doesn’t collect or store any user data directly, and we minimize the information exchanged between systems. Information about the user’s progress and credentials for interacting with the eligibility verification services are temporarily stored in an encrypted session in the user’s browser. Once enrolled, a user’s payment card information is saved policies and information on by Littlepay. Credentials / identity proofing information provided to Login.gov and payment card information provided to Littlepay are never accessible by the Benefits system/team.
```

Comment:
This part of the sentence is not clear to me:

> Once enrolled, _**a user’s payment card information is saved policies and information on by Littlepay.**_

Reply:
Ha, whoops! Took another pass and did other cleanup while I was at it.